src/providers/twitch/TwitchAccount.cpp

```diff
@@ -138,51 +140,60 @@ void TwitchAccount::loadBlocks()
         std::move(token));
 }
 
-void TwitchAccount::blockUser(const QString &userId, const QObject *caller,
+void TwitchAccount::blockUser(const QString &userId, const QString &userLogin,
+                              const QObject *caller,
                               std::function<void()> onSuccess,
                               std::function<void()> onFailure)
 {
     getHelix()->blockUser(
         userId, caller,
-        [this, userId, onSuccess = std::move(onSuccess)] {
+        [this, userId, userLogin, onSuccess = std::move(onSuccess)] {
             assertInGuiThread();
 
             TwitchUser blockedUser;
             blockedUser.id = userId;
+            blockedUser.name = userLogin;
             this->ignores_.insert(blockedUser);
             this->ignoresUserIds_.insert(blockedUser.id);
+            this->ignoresUserLogins_.insert(blockedUser.name);
             onSuccess();
         },
         std::move(onFailure));
 }
 
-void TwitchAccount::unblockUser(const QString &userId, const QObject *caller,
+void TwitchAccount::unblockUser(const QString &userId, const QString &userLogin,
+                                const QObject *caller,
                                 std::function<void()> onSuccess,
                                 std::function<void()> onFailure)
 {
     getHelix()->unblockUser(
         userId, caller,
-        [this, userId, onSuccess = std::move(onSuccess)] {
+        [this, userId, userLogin, onSuccess = std::move(onSuccess)] {
             assertInGuiThread();
 
             TwitchUser ignoredUser;
             ignoredUser.id = userId;
+            ignoredUser.name = userLogin;
             this->ignores_.erase(ignoredUser);
             this->ignoresUserIds_.erase(ignoredUser.id);
+            this->ignoresUserLogins_.erase(ignoredUser.name);
             onSuccess();
         },
         std::move(onFailure));
 }
 
-void TwitchAccount::blockUserLocally(const QString &userID)
+void TwitchAccount::blockUserLocally(const QString &userID,
+                                     const QString &userLogin)
 {
     assertInGuiThread();
     assert(getApp()->isTest());
 
     TwitchUser blockedUser;
     blockedUser.id = userID;
+    blockedUser.name = userLogin;
     this->ignores_.insert(blockedUser);
     this->ignoresUserIds_.insert(blockedUser.id);
+    this->ignoresUserIds_.insert(blockedUser.name);
```

Comment:
```suggestion
    this->ignoresUserLogins_.insert(blockedUser.name);
```

Reply:
Fixed in 59b163b91b3c2f81252a7319767bb024e1b1f518